app/src/main/AndroidManifest.xml

```diff
@@ -511,15 +511,45 @@
                 android:resource="@xml/mensa_widget_info" />
         </receiver>
 
-        <!-- receiver for the MVVWidget -->
         <service
             android:name=".services.MensaWidgetService"
             android:permission="android.permission.BIND_REMOTEVIEWS" />
 
+        <!-- receiver for the MVVWidget -->
+
+        <receiver android:name=".widgets.MVVWidget"
+            android:label="@string/mvv_widget">>
+            <intent-filter>
+                <action android:name="android.appwidget.action.APPWIDGET_UPDATE" />
+            </intent-filter>
+
+            <meta-data
+                android:name="android.appwidget.provider"
+                android:resource="@xml/mvv_widget_info" />
+        </receiver>
+
         <service
-            android:name=".widgets.CardsWidgetService"
+            android:name=".widgets.MVVWidgetService"
```

Comment:
this is just wrong behaviour in git... ive added the service for the mvv widget, the card wiged stays untouched and is behind the mvv widget in the manifest file